### DISCUSSION TITLE
Please add a feature to plot in log-scale in either X-axis or Y-axis or both in Plot

### DISCUSSION THREAD

**Discussion by jwu-edm** [sqlitebrowser/sqlitebrowser#3695] [Ideas]
I am a user of the DB Brower for SQLite. It is such a great program! When I tried to plot the computed spectrum of a seismic ground acceleration time-history record vs time or frequency, I found that it would be great to add a feature to plot in log-scale in either X-axis or Y-axis or both (selected by users) in the Plot area of the program. Thanks.

**Comment by FriedrichFroebel** [comment]
This has already been requested before #3633.
